### ISSUE TITLE
Duplicate function check not working properly for functions in different namespaces

### ISSUE THREAD

**Issue by gsf-greece** [ligasgr/intellij-xquery#188]
I couldn't find an open issue about this so I opened a new one. When you declare functions in different namespaces, in the same file, for example like this:

```
declare namespace test1 = "test1";
declare namespace test2 = "test2";
declare function test1:test() {
  
};
declare function test2:test() {
  
};
test2:test()
```

You get an error that the function is already defined.

**Comment by ligasgr** [comment]
Hey,

can you please retest this on the last version of plugin?
I cannot reproduce the issue with code:
``` xquery
xquery version '1.0';
declare namespace test1 = "test1";
declare namespace test2 = "test2";

declare function test1:test() {
    'cab'
};

declare function test2:test() {
    'abc'
};

test2:test()
```
Can you please also let me know the OS, version of the plugin, version of Idea, which language flavour you have selected in the settings of the plugin?

Thanks

**Comment by gsf-greece** [comment]
I **can't ** reproduce this on IntelliJ 2016.1.1 on Windows 10 64 with the latest xquery plugin, and any selected flavour. I might have seen this on Linux though, I can't remember unfortunately.

I did however noticed this: https://github.com/ligasgr/intellij-xquery/issues/98 when using BaseX as a data source (the function I made to test the bug had || concatenation in it, and I happened to select BaseX), and if I create a file with name test.xquery the plugin doesn't recognize it as an xquery file.
